rega/cmd.py

```diff
@@ -254,14 +234,12 @@ def get_tf_modules(target):
 
 def terraform_plan(target, backend, config):
```

Comment:
```suggestion
def terraform_plan(target, config):
```

This should be updated on all instances.